lib/Agrammon/Web/APIRoutes.pm6

```diff
@@ -24,6 +24,12 @@ my class AgrammonAPITokenMiddleware does Cro::APIToken::Middleware {
         }
         return $request;
     }
+
+    method on-invalid(Cro::HTTP::Request $request, Cro::APIToken::Token $token) {
+        $request.target eq '/openapi.' ~ any(<yaml json>)
```

Comment:
That's some fancy syntax.

Reply:
`'/openapi.' ~ any(<yaml json>)` auto-threads over the `~`, and produces `any(<openapi.yaml openapi.json>)`, but without the repetition. :)